server/config/fusor.yaml

```diff
@@ -228,7 +240,7 @@
            :name: "OpenShift"
            :content: :openshift
            :subscription_descriptions:
-             - "Red Hat Cloud Infrastructure"
+             - "Red Hat OpenShift Enterprise"
```

Comment:
@johnkim76 don't just change it back. It won't work. I just dug through the code, it needs to be 'BOTH'.
```
            :subscription_descriptions:
              - "Red Hat Cloud Infrastructure"
              - "OpenShift Enterprise"
```

These are used to associate the subscriptions to the activation key. Without them the subscription won't be there when the activation key is used. Also note, that it is **OpenShift Enterprise** without the RH. 

I uploaded a new manifest. The **Description** field of the Subscriptions page is what is used. Also the branding fields from the candlepin database:

```
candlepin=# select id, productid, productname from cp_pool;
                id                |   productid   |                               productname                               
----------------------------------+---------------+-------------------------------------------------------------------------
 ff8080815360e1a1015360e872630005 | 1457619759260 | Fusor
 ff80808153867547015387670b850199 | RV00007       | Red Hat Cloud Infrastructure with Smart Management, Premium (2-sockets)
 ff80808153867547015387670bb401c5 | RV00007       | Red Hat Cloud Infrastructure with Smart Management, Premium (2-sockets)
 ff80808153867547015387670c7901f7 | SER0480       | OpenShift Business Partner Self-Supported NFR
(4 rows)

candlepin=# select * from cp_branding ;
                id                |          created           |          updated           | productid | type |             name             
----------------------------------+----------------------------+----------------------------+-----------+------+------------------------------
 ff80808153867547015387670a340195 | 2016-03-17 21:46:30.324-04 | 2016-03-17 21:46:30.324-04 | 69        | OS   | Red Hat Cloud Infrastructure
 ff80808153867547015387670a620198 | 2016-03-17 21:46:30.37-04  | 2016-03-17 21:46:30.37-04  | 69        | OS   | OpenShift Enterprise
 ff80808153867547015387670b85019a | 2016-03-17 21:46:30.661-04 | 2016-03-17 21:46:30.661-04 | 69        | OS   | Red Hat Cloud Infrastructure
 ff80808153867547015387670bb401cc | 2016-03-17 21:46:30.708-04 | 2016-03-17 21:46:30.708-04 | 69        | OS   | Red Hat Cloud Infrastructure
 ff80808153867547015387670c7901f8 | 2016-03-17 21:46:30.905-04 | 2016-03-17 21:46:30.905-04 | 69        | OS   | OpenShift Enterprise
(5 rows)
```